src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -0,0 +1,60 @@
+package mate.jdbc.model;
+
+import java.util.Objects;
+
+public class Driver {
+    private Long id;
+    private String name;
+    private String licenseNumber;
+
```

Comment:
```suggestion
public Driver() {
}

public Driver(String name, String licenseNumber) {
        this.name = name;
        this.licenseNumber = licenseNumber;
}

public Driver(Long id, String name, String licenseNumber) {
        this.id = id;
        this.name = name;
        this.licenseNumber = licenseNumber;
}
```

Reply:
at your discretion, you can add several variants of constructors and use them when creating instead of setters